### PR TITLE
Updating update managers error message to be more clear

### PR DESCRIPTION
**Pull request by PeriGK**
Hi,

I took the initiative to change a bit the message, as requested in #284 

Can you please have a look?

EDIT: I also noticed that there is no README file, so I added one as well.

Thanks,
P.

### CODE REVIEW

**Comment by clefebvre** [comment]
thanks @PeriGK 

2 things: 

- Don't add that README, it adds nothing here. First, the git repo could be read somewhere else than on github... and second, that information is already there in the github repo description.

- Can you rephrase "Please click the OK button above, to switch to a different Linux Mint mirror and solve this problem." into "To switch to a different Linux Mint mirror and solve this problem, click OK.".

**Comment by PeriGK** [comment]
Hi @clefebvre ,

I have done the changes, please let me know if you need anything else.

Thanks,
P.
